test/e2e/tests/ipfs-ens-resolution.spec.js

```diff
@@ -23,6 +23,10 @@ describe('Settings', function () {
       // since all we care about is getting to the correct URL
     }
 
+    // Provide time for the browser to error, send the the error to
+    // browser.webRequest.onErrorOccurred, and our system to redirect
+    await driver.delay(largeDelayMs);
+
     // Ensure that the redirect to ENS Domains has happened
     const currentUrl = await driver.getCurrentUrl();
```

Comment:
Nit: Perhaps we could use a `driver.wait` loop here to check the URL periodically until it matches what we expect? That would let it exit early if the URL changes sooner.